fastlane/Fastfile

```
@@ -0,0 +1,119 @@
+      release_notes_file: "FirebaseAppDistributionConfig/release_notes.txt",
+      groups_file: "FirebaseAppDistributionConfig/groups.txt",
+      debug: true
+    )
+  end
+
+  desc "Build, Sign, and Distribute Release APK to Play Store"
+  lane :build_and_distribute_release do |options|
+    keystore_path = File.expand_path('../keystore.jks')
+    package_name = get_package_name(options[:variant])
+    new_version_code = latest_googleplay_version_code(package_name) + 1
+    current_version_name = android_get_version_name
```

rubocop (1.69.1)

[warning] Useless assignment to variable - `current_version_name`. Did you mean `new_version_name`?

(Lint/UselessAssignment)